plugins/vpc-eni/network/network_windows.go

```diff
@@ -198,6 +201,53 @@ func (nb *NetBuilder) FindOrCreateEndpoint(nw *Network, ep *Endpoint) error {
 		hnsEndpoint.PrefixLength = uint8(pl)
 	}
 
+	// Add ACL policies for blocking IMDS access through the endpoint.
+	if ep.BlockIMDS {
+		// Create an ACL policy to block traffic to instance metadata endpoint.
+		err = nb.addEndpointPolicy(hnsEndpoint,
```

Comment:
Put the first arg (hnsEndpoint) in its own line. Multiple occurrences in this block.